pkgs/by-name/ob/obj2voxel/package.nix

```
@@ -0,0 +1,73 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, fetchpatch
+, cmake
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "obj2voxel";
+  version = "1.3.4";
+
+  src = (
+    fetchFromGitHub {
+      owner = "Eisenwave";
+      repo = finalAttrs.pname;
```

```suggestion
      repo = "obj2voxel";
```
If we append something to pname, that would no longer be true

---

Doesn't sound unreasonable, but wouldn't it be more appropriate to apply this change using a treewide commit supplemented by some contributor documentation to be read by all contributors?

I'm asking because using `pname` to construct repo URLs seems to be very common even among new packages, as verified by doing
```
grep -r 'repo = .*pname' pkgs/by-name
```
(`pkgs/by-name` being a fairly new mechanism)